mgmt_cli_test.py

```diff
@@ -546,8 +547,9 @@ def test_restore_backup_with_task(self):
         backup_task_status = backup_task.wait_and_get_final_status(timeout=1500)
         assert backup_task_status == TaskStatus.DONE, \
             f"Backup task ended in {backup_task_status} instead of {TaskStatus.DONE}"
-        self.verify_backup_success(mgr_cluster=mgr_cluster, backup_task=backup_task, restore_data_with_task=True,
-                                   timeout=3000)
+        timeout = 36 * 60
+        with adaptive_timeout(Operations.MGMT_REPAIR, self.db_cluster.nodes[0], timeout=timeout):
+            self.verify_backup_success(mgr_cluster=mgr_cluster, backup_task=backup_task, restore_data_with_task=True)
```

Comment:
Is it intentional that the `verify_backup_success` method doesn't have anymore provided `timeout` param which should have been the `hard timeout`?

Reply:
will add